packages/ui/src/validators/hooks/useStakingStatistics.tsx

```diff
@@ -21,11 +21,17 @@ export const useStakingStatistics = () => {
 
   const now = useObservable(() => api?.query.timestamp.now(), [api?.isConnected])
   const totalIssuance = useObservable(() => api?.query.balances.totalIssuance(), [api?.isConnected])
-  const currentStaking = useObservable(() => activeEra ? api?.query.staking.erasTotalStake(activeEra.eraIndex):undefined, [activeEra, api?.isConnected])
+  const currentStaking = useObservable(
+    () => activeEra && api?.query.staking.erasTotalStake(activeEra.eraIndex),
+    [activeEra, api?.isConnected]
+  )
   const activeValidators = useObservable(() => api?.query.session.validators(), [api?.isConnected])
   const stakers = useObservable(() => {
     if (activeValidators && api)
-      return  activeEra ? combineLatest(activeValidators.map((address) => api.query.staking.erasStakers(activeEra.eraIndex, address))): undefined
+      return (
+        activeEra &&
```

Comment:
I have modified the code according to your review so that the storybook seems to  be good now.
And in the Piechart, `100%` is never displayed in any case.